releases/release-1.31/release-notes/maps/pr-121970-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 121970
+releasenote:
+  text: 'k8s.io/apimachinery/pkg/util/runtime: new calls support handling panics and
```

```suggestion
  text: 'k8s.io/apimachinery/pkg/util/runtime: Added support for new calls to handle panics and
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653153392